app/policies/user_policy.rb

```diff
@@ -0,0 +1,38 @@
+class UserPolicy < ApplicationPolicy
+  attr_reader :current_user, :user
+
+  def initialize(current_user, user)
+    @current_user = current_user
+    @user = user
+  end
+
+  def show?
+    true
+  end
+
+  def feed?
+    true
+  end
+
+  def edit?
+    update?
+  end
+
+  def update?
```

Comment:
Do we need to give users edit and update access based on the current version of the photogram app? Also, nitpick; if you are going to use inheritance, place update? before edit?